app/cloud/index.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Kong Cloud
+layout: default
+hide_subscribe: true
+id: cloud
+---
+
+<section class="section enterprise-section">
+  <div class="container">
+    <div class="row">
+      <div class="one-half column">
+        <h2>The Power of Kong, now delivered SaaS</h2>
+        <p>Kong Cloud comes with high-performance hosting, SLA, Backups, HA, 24x7 Support and more.</p>
+        <p>
+          <a href="#request-demo" class="button button-primary button-large scroll-to">Request Demo</a>
+        </p>
+      </div>
+      <div class="one-half column">
+        <figure>
+          <img src="/assets/images/enterprise/enterprise.svg" width="440" height="222" alt="Enterprise"/>
+        </figure>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section demo-request-section" id="request-demo">
+  <div class="container">
+    <header class="section-header">
+      <h2>START YOUR KONG CLOUD</h2>
+      <p>You're Just a Few Clicks Away</p>
+    </header>
+
+    <form class="demo-request-form">
```

Comment:
Since this form uses `.demo-request-form`, its expected behaviour is to follow [app.js#L112](https://github.com/Mashape/getkong.org/blob/fdb06f710617e4f1a2d120ddb3dbe4c28184c57f/app/_assets/javascripts/app.js#L112) - but was not tested.